eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/NavLink.js

```diff
@@ -85,11 +97,11 @@ const NavLink = ({
       <Title>{children}</Title>
     </IconText>
 
-    {isSection && isOpen && errorCount > 0 ? (
-      <Badge data-test="badge-NoCount-open" />
+    {isSection && !isOpen && (sectionTotalErrors || errorCount) !== 0 ? (
+      <SmallBadge data-test="badge-NoCount-closed" />
     ) : null}
-    {isSection && !isOpen && (sectionTotalErrors || errorCount) > 0 ? (
-      <Badge data-test="badge-NoCount-closed" />
+    {isSection && isOpen && errorCount > 0 && errorCount !== 0 ? (
```

Comment:
duplicate in statement?
could be:
{isSection && isOpen && errorCount !== 0 ? (
or just:
{isSection && isOpen && && errorCount !== 0 ? (

Reply:
Woowaa